mobify/progressive-web-scaffold
# Astro: Cart Header Badge Counter

(brief description / PR title)

 **JIRA**: https://mobify.atlassian.net/browse/HYB-984
 **Linked PRs**: https://github.com/mobify/progressive-web-scaffold/pull/229

## Changes
- on web, added an event that triggers on cart counter changes
- on Native side, implemented badge controller that handles these changes

## How to test-drive this PR
- start app, add items to cart
- go back to home screen (The cart is not on all pages yet waiting to be fixed)
- should see the changes on the badge
